docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,392 @@
+
+=== Kubernetes
+
+At Keep we run on GCP + Kube. To accommodate the aforementioned constraints and system
+recommendations we use the following pattern for each of our environments:
+
+- Regional Kube cluster.
+- 5 beacon clients, each running minimum stake required by the network.
+- A LoadBalancer Service for each client.
+- A StatefulSet for each client.
+
+You can see our Ropsten Kube configurations https://github.com/keep-network/keep-core/tree/master/infrastructure/kube/keep-test[here]
+
+== Logging
+
+Below are some of the key things to look out for to make sure you're booted and connected to the
+network:
+
+=== Startup
+```
+▓▓▌ ▓▓ ▐▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓    ▓▓▓▓▓▓▓▀    ▐▓▓▓▓▓▓    ▐▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▄▄▓▓▓▓▓▓▓▀      ▐▓▓▓▓▓▓▄▄▄▄         ▓▓▓▓▓▓▄▄▄▄         ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▓▓▓▓▓▓▓▀        ▐▓▓▓▓▓▓▓▓▓▓         ▓▓▓▓▓▓▓▓▓▓▌        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓▀▀▓▓▓▓▓▓▄       ▐▓▓▓▓▓▓▀▀▀▀         ▓▓▓▓▓▓▀▀▀▀         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▀
+  ▓▓▓▓▓▓   ▀▓▓▓▓▓▓▄     ▐▓▓▓▓▓▓     ▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌
+▓▓▓▓▓▓▓▓▓▓ █▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+
+Trust math, not hardware.
+
+-----------------------------------------------------------------------------------------------
+| Keep Random Beacon Node                                                                     |
+|                                                                                             |
+| Port: 3919                                                                                  |
+| IPs : /ip4/127.0.0.1/tcp/3919/ipfs/16Uiu2HAmCcfVpHwfBKNFbQuhvGuFXHVLQ65gB4sJm7HyrcZuLttH    |
+|       /ip4/10.102.0.112/tcp/3919/ipfs/16Uiu2HAmCcfVpHwfBKNFbQuhvGuFXHVLQ65gB4sJm7HyrcZuLttH |
+-----------------------------------------------------------------------------------------------
+```
+
+**Bonus**: If you want to share your LibP2P address with others you can get it from the startup log.  When sharing remember to substitute the `/ipv4/` address with the public facing IP of your client if you're running on a private machine, or replace the entire `/ipv4/` segment with a DNS entry if you're using a hostname.
+
+=== Peer Connections
+
+```
+21:19:47.129 DEBUG keep-net-w: connected to [1] peers:[16Uiu2HAm3eJtyFKAttzJ85NLMromHuRg4yyum3CREMf6CHBBV6KY]
+```
+
+== ETH Networks
+
+=== Mainnet
+
+==== Boostrap Peers
+
+==== Contracts
+
+[%header,cols=2*]
+|===
+|Token
+|
+
+|KeepToken
+|''
+
+|TokenStaking
+|''
+
+|TokenGrant
+|''
+|===
+
+[%header,cols=2*]
+|===
+|Group
+|
+
+|Groups
+|''
+
+|GroupSelection
+|''
+|===
+
+[%header,cols=2*]
+|===
+|RandomBeacon
+|
+
+|DKGResultVerification
+|''
+
+|KeepRandomBeaconServiceImplV1
+|''
+
+|KeepRandomBeaconService
+|''
+
+|KeepRandomBeaconOperator
+|''
+|===
```

Comment:
From operator's perspective there are only two contracts that matter: `KeepRandomBeaconOperator` and `TokenStaking`.

Reply:
Are you saying we should remove everything but those?

Reply:
Probably yes, though we can wait for another PR.

We also probably want to pull these in via a CSV (see [the asciidoctor docs](https://asciidoctor.org/docs/user-manual/#delimiter-separated-values)) so it's straightforward to generate programmatically.